### PR TITLE
Phil spel/exp imp solver demonstrator

### LINKED ISSUE #148: Simplify euler-stability-demonstrator
- [x] more flexibility in implicit solver for different functions
- [x] nicer comments in code

### PR DESCRIPTION
**Pull request by PhiSpel**
checklist in issue!

closes #148

### CODE REVIEW

**Review by sandro-elsweijer — CHANGES_REQUESTED** [review]

File: content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb [hunk 1]
```diff
@@ -6,232 +6,391 @@
    "source": [
     "# Stabilität Eulerverfahren\n",
     "\n",
-    "Bestimmte DGL lassen sich mit impliziten Verfahren besser lösen als mit expliziten Verfahren. Diese DGL bezeichnen wir als **steif**.\n",
+    "Bestimmte ODEs lassen sich mit impliziten Verfahren besser lösen als mit expliziten Verfahren. Diese ODEs bezeichnen wir als **steif**.\n",
     "\n",
-    "Das liegt vor allem an der hohen Lipschitzkonstante, die für explizite Verfahren eine sehr kleine Schrittweite bedeutet $\\left(h<\\frac{2}{\\lambda}\\right)$."
+    "Das liegt vor allem an der hohen Lipschitzkonstante, die für explizite Verfahren eine sehr kleine Schrittweite bedeutet $\\left(h<\\frac{2}{|\\lambda|}\\right)$."
```
[the remark this answers is not in the document]
Comment (PhiSpel) [reply content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb]: Added a little bit more context and explained that h_max is easily determined for e^\lambda*t
See dfe386a

File: content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb [hunk 2]
```diff
@@ -6,232 +6,391 @@
...
+    "## Aufgabe 2\n",
     "\n",
-    "- Wie ändert sich die Lösung?\n",
-    "- Warum ist `h_selection = [0.5 * Lip, 1.01 * Lip];` abhängig von `lambda` gewählt? Was passiert, wenn `h` unabhängig von `lambda` gewählt wird? Was, wenn `h` für das explizite Verfahren nur minimal kleiner als $\\frac{2}{|\\lambda|}$ ist?\n",
-    "- Wann ist das explizite Verfahren nicht mehr hilfreich? Wie 'steif' sieht die zugehörige Funktion aus?"
+    "Variieren Sie nun mithilfe der App aus dem unten stehenden Code die verfügbaren Parameter sowie die Funtionen für $g$. Gegebenenfalls müssen Sie den Code lokal ausführen.\n",
+    "\n",
+    "- Wie unterscheiden sich die Lösungen beider Verfahren?\n",
+    "- Was passiert, wenn $h$ für das explizite Verfahren nur minimal kleiner als $\\frac{2}{|\\lambda|}$ ist? Was passiert, wenn $h$ größer ist? Wo liegt der Unterschied zum impliziten Verfahren?\n",
+    "- Bei welchen Funktionen ist das explizite Verfahren nicht mehr hilfreich? Was macht die'Steifheit' dieser Funktionen qualitativ aus?\n",
```
[the remark this answers is not in the document]
Comment (PhiSpel) [reply content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb]: See dfe386a

File: content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (sandro-elsweijer) [reply content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb]: The part after "Mathematisch ausgedrückt:" seems weird, maybe we should give the formula its own line and refine the grammar a little.

[the remark this answers is not in the document]
Comment (PhiSpel) [reply content/07_num_ode/02_stabilitaet_eulerverfahren.ipynb]: see ba54f47

**Pull request merged**